package.json

```diff
@@ -13,6 +13,7 @@
     "test": "mocha -r ts-node/register 'src/**/*.test.ts' && yarn lint",
     "lint": "xo",
     "build": "tsc -p ./tsconfig.json && uglifyjs build/index.js -o build/index.js -m -c",
+    "prepare": "npm run build",
```

Comment:
```suggestion
    "prepare": "yarn build",
```

Reply:
When scripts do stuff it should be included in the devDependencies. `yarn` is currently not included there. Shall I change that?

Reply:
We should probably include that besides Node.js, Yarn needs to be installed as well. Installing Yarn via npm isn't [recommended](https://stackoverflow.com/a/43377805) and it might conflict with the system-wide install vs. the one in `node_modules`.

Or we just keep `npm run build`, I don't really mind that much.

Reply:
I prefer to have repositories self contained. They shouldn't need something specific installed globally in my opinion. Especially as dependency versions might differ in different repositories.